content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -26,18 +26,17 @@ You can use either `kubectl create configmap` or a ConfigMap generator in `kusto
 
 ### Create a ConfigMap Using kubectl create configmap
 
-Use the `kubectl create configmap` command to create configmaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
+Use the `kubectl create configmap` command to create ConfigMaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
 
 ```shell
 kubectl create configmap <map-name> <data-source>
 ```
 
 where \<map-name> is the name you want to assign to the ConfigMap and \<data-source> is the directory, file, or literal value to draw the data from.
 
-The data source corresponds to a key-value pair in the ConfigMap, where
+When you are creating a ConfigMap based on a file, the key in the \<data-source> defaults to the basename of the file, and the value defaults to the file content.
 
-* key = the file name or the key you provided on the command line, and
-* value = the file contents or the literal value you provided on the command line.
+When you are creating a ConfigMap based on a directory, kubectl identifies files whose basename is a valid key in the directory and packages each of those files into the new ConfigMap. Any directory entries except regular files are ignored (e.g. subdirectories, symlinks, devices, pipes, etc).
```

Comment:
Does this fit better in the section headed “Create ConfigMaps from directories”? (I think it does).

I can see that it's important to let readers know that there's more detail to read up on if they do this.

Reply:
It does. My original intent was to rewrite these paragraphs so that they are easier to read. If we move it to the "Create ConfigMaps from directories" should we also move the paragraph about files to the relevant file section as well just to be consistent.

Reply:
I have made the changes.